Sources/LiveViewNative/LiveView.swift

```diff
@@ -45,18 +45,15 @@ public macro LiveView<Host: LiveViewHost>(
 public struct LiveView<R: RootRegistry>: View {
     @StateObject var session: LiveSessionCoordinator<R>
     
-    @ObservedObject private var rootCoordinator: LiveViewCoordinator<R>
-    
     @StateObject private var liveViewModel = LiveViewModel()
     
     @Environment(\.scenePhase) private var scenePhase
     
     /// Creates a new LiveView attached to the given coordinator.
     ///
     /// - Note: Changing coordinators after the `LiveView` is setup and connected is forbidden.
-    public init(session: LiveSessionCoordinator<R>) {
-        self._session = .init(wrappedValue: session)
-        self.rootCoordinator = session.navigationPath.first!.coordinator
+    public init(session: @autoclosure @escaping () -> LiveSessionCoordinator<R>) {
```

Comment:
I've made this take a closure instead, which should help with embedding a `LiveView` in an app with state that changes outside.

It's marked `@autoclosure` so the API looks the same as before.